static/templates/admin_linkifier_list.hbs

```diff
@@ -11,6 +11,9 @@
         <button class="button small delete btn-danger" data-linkifier-id="{{id}}">
             <i class="fa fa-trash-o" aria-hidden="true"></i>
         </button>
+        <button class="button small edit btn-warning" data-linkifier-id="{{id}}">
+            <i class="fa fa-pencil" aria-hidden="true"></i>
```

Comment:
This should be `aria-label` (with a title and tippy tooltip), not `aria-hidden`, sine the button does not have a textual label.

(`aria-hidden` is only for icons on buttons that have a plain-text label too, e.g. in our gear menu).

Reply:
(The adjacent button is broken the same way; can you fix that in a prep commit?)